core/runtime/binaryen/runtime_external_interface.cpp

```diff
@@ -154,6 +167,33 @@ namespace kagome::runtime::binaryen {
                                                   arguments.at(2).geti32());
         return wasm::Literal(res);
       }
+      /// ext_default_child_storage_clear_version_1
+      if (import->base == ext_default_child_storage_clear_version_1) {
+        return wasm::Literal();
```

Comment:
I believe it's better to explicitly state somewhere that a method is not implemented. In logs, using a [deprecated] attribute or something like this.